tests/tt_metal/tt_metal/unit_tests_common/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ set(UNIT_TESTS_COMMON_SRC
     ${CMAKE_CURRENT_SOURCE_DIR}/watcher/test_waypoint.cpp
 )
 add_library(unit_tests_common_o OBJECT ${UNIT_TESTS_COMMON_SRC})
-target_link_libraries(unit_tests_common_o PUBLIC compiler_flags metal_header_directories)
+target_link_libraries(unit_tests_common_o PUBLIC compiler_flags metal_header_directories umd_device gtest gtest_main)
```

Comment:
Do we need `umd_device` here?

Reply:
Yep, for the same reason as the other cases...